components/Button/Button.js

```diff
@@ -0,0 +1,17 @@
+import { StyledButton, StyledPrimaryButton } from "./Button.styled";
+
+export default function Button({ children, type, onClick }) {
+  return (
+    <StyledButton type={type} onClick={onClick}>
+      {children}
+    </StyledButton>
+  );
+}
+
+export function PrimaryButton({ children, type, onClick }) {
+  return (
+    <StyledPrimaryButton type={type} onClick={onClick}>
+      {children}
+    </StyledPrimaryButton>
```

Comment:
this is basically the same component, with different styling. 
you could think about a porp, that tells if the Button is a primary or not and adust the styling accordingly.

Reply:
I changed it to be just one component with an if condition and a prop called "buttonType". 

```
export default function Button({ children, type, onClick, buttonType }) {
  if (buttonType === "primary") {
    return (
      <StyledPrimaryButton type={type} onClick={onClick}>
        {children}
      </StyledPrimaryButton>
    );
  }

  return (
    <StyledButton type={type} onClick={onClick}>
      {children}
    </StyledButton>
  );
}
```

@julialoeschel: Is this what you were thinking of?